src/I18nBundle/DependencyInjection/Configuration.php

```diff
@@ -143,6 +144,30 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->treatNullLike(['enabled' => false])
                     ->end()
                 ->end()
+                ->arrayNode('cookie')
+                    ->info('Cookie settings')
```

Comment:
Please add the `addDefaultsIfNotSet()`. This new cookie section is not available in most updates/ new installation, so we need always to provide the default values (tests are also failing because of this)

Reply:
Good catch! Fixed 😃